### PR TITLE
Make sure Bumpdown is compatible with the new JS

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#62321](https://central.tri.be/issues/62321)
_Rel:_ moderntribe/tribe-common#114

### CODE REVIEW

File: src/admin-views/rsvp-metabox-advanced.php [hunk 1]
```diff
@@ -9,22 +9,19 @@
 
 <?php
 if ( class_exists( 'Tribe__Events__Pro__Main' ) ) {
+	if ( is_admin() ) {
+		$bumpdown = __( 'Currently, Tickets will only show up on the frontend once per full event. For PRO users this means the same ticket will appear across all events in the series. Please configure your events accordingly.', 'event-tickets' );
```
Comment (barryhughes) [inline src/admin-views/rsvp-metabox-advanced.php]: Feels like _'Tickets'_ shouldn't be capitalized here (same for line 15).

Comment (barryhughes) [inline src/admin-views/rsvp-metabox-advanced.php]: Oh, I see - this is pre-existing text...

**Comment by barryhughes** [comment]
Tagging merge, could optionally adjust the text re my notes but not essential here.

File: src/admin-views/rsvp-metabox-advanced.php (hunk 1 above)
Comment (bordoni) [inline src/admin-views/rsvp-metabox-advanced.php]: Ha! @zbtirrell can you pitch in here?

Comment (zbtirrell) [inline src/admin-views/rsvp-metabox-advanced.php]: it should be lowercase: "tickets"

Comment (borkweb) [inline src/admin-views/rsvp-metabox-advanced.php]: If I get a vote, I'd suggest: TiCkEtS

**Pull request merged**